### PR TITLE
Django App Setup

### PR DESCRIPTION
**Pull request by aggarwal-h**
This PR resolves https://github.com/HKanwal/kapstone/issues/18.

### CODE REVIEW

File: src/django-app/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Setup Instructions
+
+## Step 1: Create a Postgres DB
+- Install PostgreSQL (if not already installed)
+- On the terminal, run:
+```bash
+psql
```
Comment (tevisdoe) [inline src/django-app/README.md:7]: When first running this you will have to run the command `psql -U postgres` to login as the postgres user so you can set things up. I don't know if you want to go into installing postgres too much, but maybe just make this change to make installation smoother for first timers:

```suggestion
psql -U postgres
```

File: src/django-app/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Step 2: Install Python Packages
+
+- Create a virtual env:
+
+  - Run:
+
+    - `pip3 install virtualenv`
+
+    - `virtualenv env`
+    - `source env/bin/activate`
```
[the remark this answers is not in the document]
Comment (aggarwal-h) [reply src/django-app/README.md]: I have updated the README with your suggestions. Thanks Tevis! 🙂

**Comment by HKanwal** [comment]
@aggarwal-h I changed the DB defaults to match the Postgres defaults (username = postgres, password = postgres, port = 5432). Please review it and let me know what you think.

**Review by HKanwal — APPROVED** [review]

**Comment by tevisdoe** [comment]
> @aggarwal-h I changed the DB defaults to match the Postgres defaults (username = postgres, password = postgres, port = 5432). Please review it and let me know what you think.
> 
> Other than that, I got it working. Looks good.

I don't really care either way, but doing this removes the need to create a new role earlier in the installation instructions. If we're going to use the postgres default user, we should remove that part of the installation instructions.

**Comment by aggarwal-h** [comment]
Thank you @HKanwal and @tevisdoe for your reviews. This PR looks good to be merged.

**Pull request merged**